src/styles/card.scss

```diff
@@ -0,0 +1,118 @@
+@use './variables.scss' as *;
+@use './stars.scss' as *;
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
```

Comment:
The grid-template-columns property uses a fixed width of 200px. Consider using a more flexible unit like percentages or fractions (fr) to make the layout more responsive to different screen sizes.